2020-10-29 18:48:40,693 [main] INFO  grakn.core.server.Grakn - Grakn started in 5.300 s
2020-10-29 19:10:53,925 [grpc-request-handler-0] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
io.grpc.StatusRuntimeException: CANCELLED: cancelled before receiving half close
	at io.grpc.Status.asRuntimeException(Status.java:524)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:275)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closedInternal(ServerCallImpl.java:353)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closed(ServerCallImpl.java:341)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed.runInContext(ServerImpl.java:864)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-29 19:47:06,279 [grpc-request-handler-0] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: email doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: email doesn't have an 'isa', a 'sub' or an 'id'
2020-10-29 19:48:55,199 [grpc-request-handler-2] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: email doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: email doesn't have an 'isa', a 'sub' or an 'id'
2020-10-29 19:52:37,220 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
2020-10-29 19:52:52,991 [grpc-request-handler-3] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
2020-10-29 19:53:41,035 [grpc-request-handler-5] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: email doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: email doesn't have an 'isa', a 'sub' or an 'id'
2020-10-29 23:27:21,681 [grpc-request-handler-3] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
2020-10-29 23:34:54,235 [grpc-request-handler-6] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
2020-10-29 23:56:15,904 [grpc-request-handler-1] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
2020-10-29 23:59:43,529 [grpc-request-handler-0] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 00:09:54,445 [main] INFO  grakn.core.server.Grakn - Grakn started in 1.997 s
2020-10-30 00:11:31,522 [grpc-request-handler-2] ERROR grakn.core.server.rpc.SessionService - An error has occurred
grakn.core.kb.server.exception.SessionException: The client session with session ID [phone_callsdf130dde-cb7b-4366-8305-fa7c59e5c881] was not found. Create a new session to interact with the graph.
	at grakn.core.kb.server.exception.SessionException.sessionNotFound(SessionException.java:36)
	at grakn.core.server.rpc.SessionService.close(SessionService.java:139)
	at grakn.protocol.session.SessionServiceGrpc$MethodHandlers.invoke(SessionServiceGrpc.java:343)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-30 00:11:49,716 [grpc-request-handler-3] ERROR g.core.server.rpc.KeyspaceService - Exception during keyspace deletion.
java.lang.RuntimeException: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:85)
	at grakn.core.server.keyspace.KeyspaceManager.delete(KeyspaceManager.java:60)
	at grakn.core.server.rpc.KeyspaceService.delete(KeyspaceService.java:60)
	at grakn.protocol.keyspace.KeyspaceServiceGrpc$MethodHandlers.invoke(KeyspaceServiceGrpc.java:281)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at com.datastax.oss.driver.api.core.DriverTimeoutException.copy(DriverTimeoutException.java:34)
	at com.datastax.oss.driver.internal.core.util.concurrent.CompletableFutures.getUninterruptibly(CompletableFutures.java:149)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:53)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:30)
	at com.datastax.oss.driver.internal.core.session.DefaultSession.execute(DefaultSession.java:208)
	at com.datastax.oss.driver.api.core.CqlSession.execute(CqlSession.java:51)
	at grakn.core.graph.diskstorage.cql.CQLStoreManager.clearStorage(CQLStoreManager.java:363)
	at grakn.core.graph.diskstorage.Backend.clearStorage(Backend.java:317)
	at grakn.core.graph.core.JanusGraphFactory.drop(JanusGraphFactory.java:97)
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:83)
	... 11 common frames omitted
2020-10-30 00:11:59,949 [grpc-request-handler-5] ERROR g.core.server.rpc.KeyspaceService - Exception during keyspace deletion.
java.lang.RuntimeException: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:85)
	at grakn.core.server.keyspace.KeyspaceManager.delete(KeyspaceManager.java:60)
	at grakn.core.server.rpc.KeyspaceService.delete(KeyspaceService.java:60)
	at grakn.protocol.keyspace.KeyspaceServiceGrpc$MethodHandlers.invoke(KeyspaceServiceGrpc.java:281)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at com.datastax.oss.driver.api.core.DriverTimeoutException.copy(DriverTimeoutException.java:34)
	at com.datastax.oss.driver.internal.core.util.concurrent.CompletableFutures.getUninterruptibly(CompletableFutures.java:149)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:53)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:30)
	at com.datastax.oss.driver.internal.core.session.DefaultSession.execute(DefaultSession.java:208)
	at com.datastax.oss.driver.api.core.CqlSession.execute(CqlSession.java:51)
	at grakn.core.graph.diskstorage.cql.CQLStoreManager.clearStorage(CQLStoreManager.java:363)
	at grakn.core.graph.diskstorage.Backend.clearStorage(Backend.java:317)
	at grakn.core.graph.core.JanusGraphFactory.drop(JanusGraphFactory.java:97)
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:83)
	... 11 common frames omitted
2020-10-30 00:12:36,971 [grpc-request-handler-6] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 00:23:56,894 [grpc-request-handler-5] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 00:24:26,811 [grpc-request-handler-4] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
io.grpc.StatusRuntimeException: CANCELLED: cancelled before receiving half close
	at io.grpc.Status.asRuntimeException(Status.java:524)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:275)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closedInternal(ServerCallImpl.java:353)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closed(ServerCallImpl.java:341)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed.runInContext(ServerImpl.java:864)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-30 00:24:38,119 [grpc-request-handler-2] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: name doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 00:46:58,170 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: age doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: age doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 00:54:27,906 [grpc-request-handler-1] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: duration doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: duration doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 01:04:48,210 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: started-at doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: started-at doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 01:06:01,387 [grpc-request-handler-3] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: started-at doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: started-at doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 01:07:01,812 [grpc-request-handler-4] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: started-at doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: started-at doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 01:08:42,165 [grpc-request-handler-6] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: duration doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: duration doesn't have an 'isa', a 'sub' or an 'id'
2020-10-30 01:44:29,238 [grpc-request-handler-0] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
	at grakn.core.concept.impl.AttributeTypeImpl.putAttribute(AttributeTypeImpl.java:136)
	at grakn.core.concept.impl.AttributeTypeImpl.create(AttributeTypeImpl.java:122)
	at grakn.core.graql.executor.ConceptBuilderImpl.putInstance(ConceptBuilderImpl.java:375)
	at grakn.core.graql.executor.ConceptBuilderImpl.tryPutConcept(ConceptBuilderImpl.java:258)
	at grakn.core.graql.executor.ConceptBuilderImpl.build(ConceptBuilderImpl.java:182)
	at grakn.core.graql.executor.WriteExecutorImpl.buildConcept(WriteExecutorImpl.java:269)
	at grakn.core.graql.executor.WriteExecutorImpl.getConcept(WriteExecutorImpl.java:406)
	at grakn.core.graql.executor.property.HasAttributeExecutor$InsertHasAttribute.execute(HasAttributeExecutor.java:96)
	at grakn.core.graql.executor.WriteExecutorImpl.write(WriteExecutorImpl.java:241)
	at grakn.core.graql.executor.WriteExecutorImpl.write(WriteExecutorImpl.java:230)
	at grakn.core.graql.executor.QueryExecutorImpl.insert(QueryExecutorImpl.java:208)
	at grakn.core.server.session.TransactionImpl.stream(TransactionImpl.java:426)
	at grakn.core.server.session.TransactionImpl.stream(TransactionImpl.java:697)
	at grakn.core.server.rpc.SessionService$TransactionListener.query(SessionService.java:388)
	at grakn.core.server.rpc.SessionService$TransactionListener.handleIterRequest(SessionService.java:278)
	at grakn.core.server.rpc.SessionService$TransactionListener.handleRequest(SessionService.java:237)
	at grakn.core.server.rpc.SessionService$TransactionListener.lambda$onNext$1(SessionService.java:196)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	... 3 common frames omitted
2020-10-30 01:45:42,494 [grpc-request-handler-5] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
	at grakn.core.concept.impl.AttributeTypeImpl.putAttribute(AttributeTypeImpl.java:136)
	at grakn.core.concept.impl.AttributeTypeImpl.create(AttributeTypeImpl.java:122)
	at grakn.core.graql.executor.ConceptBuilderImpl.putInstance(ConceptBuilderImpl.java:375)
	at grakn.core.graql.executor.ConceptBuilderImpl.tryPutConcept(ConceptBuilderImpl.java:258)
	at grakn.core.graql.executor.ConceptBuilderImpl.build(ConceptBuilderImpl.java:182)
	at grakn.core.graql.executor.WriteExecutorImpl.buildConcept(WriteExecutorImpl.java:269)
	at grakn.core.graql.executor.WriteExecutorImpl.getConcept(WriteExecutorImpl.java:406)
	at grakn.core.graql.executor.property.HasAttributeExecutor$InsertHasAttribute.execute(HasAttributeExecutor.java:96)
	at grakn.core.graql.executor.WriteExecutorImpl.write(WriteExecutorImpl.java:241)
	at grakn.core.graql.executor.WriteExecutorImpl.write(WriteExecutorImpl.java:230)
	at grakn.core.graql.executor.QueryExecutorImpl.insert(QueryExecutorImpl.java:208)
	at grakn.core.server.session.TransactionImpl.stream(TransactionImpl.java:426)
	at grakn.core.server.session.TransactionImpl.stream(TransactionImpl.java:697)
	at grakn.core.server.rpc.SessionService$TransactionListener.query(SessionService.java:388)
	at grakn.core.server.rpc.SessionService$TransactionListener.handleIterRequest(SessionService.java:278)
	at grakn.core.server.rpc.SessionService$TransactionListener.handleRequest(SessionService.java:237)
	at grakn.core.server.rpc.SessionService$TransactionListener.lambda$onNext$1(SessionService.java:196)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	... 3 common frames omitted
2020-10-30 01:48:59,143 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
	at grakn.core.concept.impl.AttributeTypeImpl.putAttribute(AttributeTypeImpl.java:136)
	at grakn.core.concept.impl.AttributeTypeImpl.create(AttributeTypeImpl.java:122)
	at grakn.core.graql.executor.ConceptBuilderImpl.putInstance(ConceptBuilderImpl.java:375)
	at grakn.core.graql.executor.ConceptBuilderImpl.tryPutConcept(ConceptBuilderImpl.java:258)
	at grakn.core.graql.executor.ConceptBuilderImpl.build(ConceptBuilderImpl.java:182)
	at grakn.core.graql.executor.WriteExecutorImpl.buildConcept(WriteExecutorImpl.java:269)
	at grakn.core.graql.executor.WriteExecutorImpl.getConcept(WriteExecutorImpl.java:406)
	at grakn.core.graql.executor.property.HasAttributeExecutor$InsertHasAttribute.execute(HasAttributeExecutor.java:96)
	at grakn.core.graql.executor.WriteExecutorImpl.write(WriteExecutorImpl.java:241)
	at grakn.core.graql.executor.WriteExecutorImpl.write(WriteExecutorImpl.java:230)
	at grakn.core.graql.executor.QueryExecutorImpl.insert(QueryExecutorImpl.java:208)
	at grakn.core.server.session.TransactionImpl.stream(TransactionImpl.java:426)
	at grakn.core.server.session.TransactionImpl.stream(TransactionImpl.java:697)
	at grakn.core.server.rpc.SessionService$TransactionListener.query(SessionService.java:388)
	at grakn.core.server.rpc.SessionService$TransactionListener.handleIterRequest(SessionService.java:278)
	at grakn.core.server.rpc.SessionService$TransactionListener.handleRequest(SessionService.java:237)
	at grakn.core.server.rpc.SessionService$TransactionListener.lambda$onNext$1(SessionService.java:196)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	... 3 common frames omitted
2020-10-30 02:00:16,702 [grpc-request-handler-1] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: label 'person' not found
2020-10-30 02:00:52,832 [grpc-request-handler-4] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: customer is not a type and so does not have instances
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: customer is not a type and so does not have instances
2020-10-30 02:02:06,727 [grpc-request-handler-3] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
io.grpc.StatusRuntimeException: CANCELLED: cancelled before receiving half close
	at io.grpc.Status.asRuntimeException(Status.java:524)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:275)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closedInternal(ServerCallImpl.java:353)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closed(ServerCallImpl.java:341)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed.runInContext(ServerImpl.java:864)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-30 02:02:18,561 [grpc-request-handler-1] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
io.grpc.StatusRuntimeException: CANCELLED: cancelled before receiving half close
	at io.grpc.Status.asRuntimeException(Status.java:524)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:275)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closedInternal(ServerCallImpl.java:353)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closed(ServerCallImpl.java:341)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed.runInContext(ServerImpl.java:864)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-31 20:37:46,770 [main] WARN  grakn.core.server.util.PIDManager - Pid file already exists: '[C:\Users\kunja\AppData\Local\Temp\grakn-core-server.pid]'. Overwriting...
2020-10-31 20:37:50,610 [main] INFO  grakn.core.server.Grakn - Grakn started in 3.332 s
2020-10-31 20:49:44,290 [grpc-request-handler-4] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: text doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: text doesn't have an 'isa', a 'sub' or an 'id'
2020-10-31 20:54:34,296 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: text doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: text doesn't have an 'isa', a 'sub' or an 'id'
2020-10-31 21:21:54,396 [grpc-request-handler-6] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: container0 doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: container0 doesn't have an 'isa', a 'sub' or an 'id'
2020-10-31 21:27:27,972 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
io.grpc.StatusRuntimeException: CANCELLED: cancelled before receiving half close
	at io.grpc.Status.asRuntimeException(Status.java:524)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:275)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closedInternal(ServerCallImpl.java:353)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closed(ServerCallImpl.java:341)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed.runInContext(ServerImpl.java:864)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-31 21:39:26,335 [grpc-request-handler-3] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: level3 doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: level3 doesn't have an 'isa', a 'sub' or an 'id'
2020-10-31 21:41:27,571 [grpc-request-handler-3] ERROR g.core.server.rpc.KeyspaceService - Exception during keyspace deletion.
grakn.core.kb.server.exception.GraknServerException: It is not possible to delete keyspace [zeitlabs] as it does not exist.
	at grakn.core.kb.server.exception.GraknServerException.create(GraknServerException.java:60)
	at grakn.core.server.keyspace.KeyspaceManager.delete(KeyspaceManager.java:55)
	at grakn.core.server.rpc.KeyspaceService.delete(KeyspaceService.java:60)
	at grakn.protocol.keyspace.KeyspaceServiceGrpc$MethodHandlers.invoke(KeyspaceServiceGrpc.java:281)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-31 21:41:41,138 [grpc-request-handler-1] ERROR g.core.server.rpc.KeyspaceService - Exception during keyspace deletion.
java.lang.RuntimeException: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:85)
	at grakn.core.server.keyspace.KeyspaceManager.delete(KeyspaceManager.java:60)
	at grakn.core.server.rpc.KeyspaceService.delete(KeyspaceService.java:60)
	at grakn.protocol.keyspace.KeyspaceServiceGrpc$MethodHandlers.invoke(KeyspaceServiceGrpc.java:281)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at com.datastax.oss.driver.api.core.DriverTimeoutException.copy(DriverTimeoutException.java:34)
	at com.datastax.oss.driver.internal.core.util.concurrent.CompletableFutures.getUninterruptibly(CompletableFutures.java:149)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:53)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:30)
	at com.datastax.oss.driver.internal.core.session.DefaultSession.execute(DefaultSession.java:208)
	at com.datastax.oss.driver.api.core.CqlSession.execute(CqlSession.java:51)
	at grakn.core.graph.diskstorage.cql.CQLStoreManager.clearStorage(CQLStoreManager.java:363)
	at grakn.core.graph.diskstorage.Backend.clearStorage(Backend.java:317)
	at grakn.core.graph.core.JanusGraphFactory.drop(JanusGraphFactory.java:97)
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:83)
	... 11 common frames omitted
2020-10-31 21:47:20,343 [grpc-request-handler-4] ERROR g.core.server.rpc.KeyspaceService - Exception during keyspace deletion.
java.lang.RuntimeException: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:85)
	at grakn.core.server.keyspace.KeyspaceManager.delete(KeyspaceManager.java:60)
	at grakn.core.server.rpc.KeyspaceService.delete(KeyspaceService.java:60)
	at grakn.protocol.keyspace.KeyspaceServiceGrpc$MethodHandlers.invoke(KeyspaceServiceGrpc.java:281)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT20S
	at com.datastax.oss.driver.api.core.DriverTimeoutException.copy(DriverTimeoutException.java:34)
	at com.datastax.oss.driver.internal.core.util.concurrent.CompletableFutures.getUninterruptibly(CompletableFutures.java:149)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:53)
	at com.datastax.oss.driver.internal.core.cql.CqlRequestSyncProcessor.process(CqlRequestSyncProcessor.java:30)
	at com.datastax.oss.driver.internal.core.session.DefaultSession.execute(DefaultSession.java:208)
	at com.datastax.oss.driver.api.core.CqlSession.execute(CqlSession.java:51)
	at grakn.core.graph.diskstorage.cql.CQLStoreManager.clearStorage(CQLStoreManager.java:363)
	at grakn.core.graph.diskstorage.Backend.clearStorage(Backend.java:317)
	at grakn.core.graph.core.JanusGraphFactory.drop(JanusGraphFactory.java:97)
	at grakn.core.server.session.JanusGraphFactory.drop(JanusGraphFactory.java:83)
	... 11 common frames omitted
2020-10-31 21:50:10,572 [grpc-request-handler-2] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
java.util.concurrent.ExecutionException: grakn.core.kb.graql.exception.GraqlSemanticException: contained doesn't have an 'isa', a 'sub' or an 'id'
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at grakn.core.server.rpc.SessionService$TransactionListener.process(SessionService.java:319)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:196)
	at grakn.core.server.rpc.SessionService$TransactionListener.onNext(SessionService.java:156)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onMessage(ServerCalls.java:251)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailableInternal(ServerCallImpl.java:309)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.messagesAvailable(ServerCallImpl.java:292)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1MessagesAvailable.runInContext(ServerImpl.java:779)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: grakn.core.kb.graql.exception.GraqlSemanticException: contained doesn't have an 'isa', a 'sub' or an 'id'
2020-10-31 22:00:47,016 [grpc-request-handler-1] ERROR grakn.core.server.rpc.SessionService - An error has occurred
grakn.core.kb.server.exception.SessionException: The client session with session ID [zeitlabseb0e1e0b-d805-4591-ab63-e7fbfcece840] was not found. Create a new session to interact with the graph.
	at grakn.core.kb.server.exception.SessionException.sessionNotFound(SessionException.java:36)
	at grakn.core.server.rpc.SessionService.close(SessionService.java:139)
	at grakn.protocol.session.SessionServiceGrpc$MethodHandlers.invoke(SessionServiceGrpc.java:343)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:814)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2020-10-31 22:02:13,222 [grpc-request-handler-7] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener: 
io.grpc.StatusRuntimeException: CANCELLED: cancelled before receiving half close
	at io.grpc.Status.asRuntimeException(Status.java:524)
	at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:275)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closedInternal(ServerCallImpl.java:353)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.closed(ServerCallImpl.java:341)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed.runInContext(ServerImpl.java:864)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
